File: 이연하/4344.py
c=int(input())

for i in range(c):
    score=list(map(int,input().split()))
    avg=sum(score[1:])/score[0] #score[1:]:점수, score[0]:학생수
    count=0

    for a in score[1:]:
        if a>avg:
            count=count+1

    pct=(count/score[0])*100
    print('%.3f' %pct +'%')

File: 박기용/4344.java
import java.io.*;

public class Main {
    public static void main(String[] args) throws IOException {
        BufferedReader br = new BufferedReader(new InputStreamReader(System.in));

        int count = Integer.parseInt(br.readLine());

        for (int i = 0; i < count; i++) {
            String[] arr = br.readLine().split(" ");

            double student = Integer.parseInt(arr[0]);
            double[] score = new double[arr.length - 1];
            double sum = 0;

            for (int j = 0; j < arr.length - 1; j++) {
                score[j] = Integer.parseInt(arr[j + 1]);
                sum += Integer.parseInt(arr[j + 1]);
            }
            double average = sum / student;

            double high = 0;

            for (int j = 0; j < student; j++) {
                if (score[j] > average) {
                    high++;
                }
            }

            double highstudent = 100 * high / student;
            System.out.printf("%.3f", highstudent);
            System.out.println("%");
        }
    }
}
